ElfPatcher.cs

```diff
@@ -10,7 +10,7 @@ namespace ToVPatcher {
 	class ElfPatcher {
 		public static void PatchElf( string ebootPath, string patchDir, string outDir, string outMd5 = null, BackgroundWorker worker = null ) {
 			
-			string patcherPath = "failOverflowTools/unself";
+			string patcherPath = "eboot_tools/unself";
```

Comment:
I'd prefer if you squashed those so that the commit that introduces the Linux patcher already uses the correct tool and names.